### PR TITLE
Add PR link to release notes

### PR DESCRIPTION
**Pull request by jviau**

### CODE REVIEW

**Review by liliankasem — APPROVED** [review]

File: release_notes.md [hunk 1]
```diff
@@ -8,4 +8,4 @@
 - Update WebJobsScriptHostService to remove hardcoded sleep during application shut down (#9520)
 - Update PowerShell 7.2 Worker Version to [4.0.2974](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2974)
 - Update PowerShell 7.0 Worker Version to [4.0.2973](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2973)
-- Add support for standalone executable (ie: `dotnet build --standalone`) for out-of-proc workers in Linux Consumption.
+- Add support for standalone executable (ie: `dotnet build --standalone`) for out-of-proc workers in Linux Consumption. ([#9550](https://github.com/Azure/azure-functions-host/pull/9550))
```
Comment (liliankasem) [inline release_notes.md]: In general we just use (#9550) without the URL

**Review by surgupta-msft — APPROVED** [review]

**Comment by liliankasem** [comment]
> No pipelines are associated with this pull request.

Guessing this is another fork issue?

**Comment by jviau** [comment]
> > No pipelines are associated with this pull request.
> 
> Guessing this is another fork issue?

yeah, thought I pushed this to be a branch off this repo. Guess not.

**Pull request merged**